File: Identity/Identity/Identity/Models/Order.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Identity.Models
{
    public class Order
    {
        public int Id { get; set; }
        public int CustomerId { get; set; }
        public DateTime Date { get; set; }
        public int StatusId { get; set; }
        public Status Status { get; set; }
        public List<OrderLine> OrderLines { get; set; }
        
        public Order()
        {
            
        }
    }
}

File: AngularFastfood/src/app/Models/Order.ts
import { Accessory, ClassAccessory } from "../Models/Accessory";

export class ClassOrder{
    id: number;
    customerId: number;
    date: Date;
    statusId: number;
}

export interface Order{
    id: number;
    customerId: number;
    date: Date;
    statusId: number;   
}
